apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-card.js

```diff
@@ -57,8 +57,12 @@ function WelcomeTourCard( {
 	return (
 		<Card className="welcome-tour-card" isElevated>
 			<CardOverlayControls onDismiss={ onDismiss } onMinimize={ onMinimize } />
-			<CardMedia>
-				<img alt={ __( 'Editor Welcome Tour', 'full-site-editing' ) } src={ imgSrc } />
+			{ /* TODO: Update selector for images in @wordpress/components/src/card/styles/card-styles.js */ }
+			<CardMedia className="welcome-tour-card__media">
+				<picture>
+					{ imgSrc.mobile && <source srcSet={ imgSrc.mobile } media="(max-width: 600px)" /> }
+					<img alt={ __( 'Editor Welcome Tour', 'full-site-editing' ) } src={ imgSrc.desktop } />
+				</picture>
```

Comment:
An alternative approach is to use `<img/>` and define the `srcset` attribute. The upside is that we don't have to introduce CSS overrides and/or request component changes. 

The downside is that you have to provide a _width_ or _density descriptor_ for each image (see [MDN Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture#attr-srcset)). The width descriptor has to be the exact size of an image, which, IMHO, is not a very user-friendly solution. `<picture/>` provides a better overview as well.